pkg/windows/windows.go

```diff
@@ -404,12 +404,18 @@ func (vm *windows) RunWICDCleanup(apiServerURL string, watchNamespace string) er
 	return nil
 }
 
-func (vm *windows) Deconfigure(watchNamespace string, apiServerURL string) error {
+func (vm *windows) Deconfigure(watchNamespace string, apiServerURL string, credentials *Authentication) error {
 	vm.log.Info("deconfiguring")
 
 	if err := vm.deconfigureWICD(); err != nil {
 		return err
 	}
+	if err := vm.ensureWICDExists(credentials); err != nil {
```

Comment:
Can you add a commit before this removing the `credentials` argument. it isnt used within ensureWICDExists